common/taskana-common/src/main/resources/sql/db2/taskana_schema_update_5.10.0_to_6.3.0_db2.sql

```diff
@@ -0,0 +1,12 @@
+-- this script updates the TASKANA database schema from version 5.10.0 to version 6.3.0.
```

Comment:
Please change "6.3.0" to the current next version :)

Reply:
fixed